radix-engine/tests/proof.rs

```diff
@@ -287,7 +287,7 @@ fn cant_move_restricted_proof() {
     // Assert
     assert_eq!(
         receipt.result,
-        Err(RuntimeError::CantMoveRestrictedProof(512))
+        Err(RuntimeError::CantMoveRestrictedProof(1025))
```

Comment:
Woah, what's going on here?

Reply:
As discussed with Josh, this only affects bucket/proof ids in errors (for which we haven't introduced ID mapping yet).